src/service/blockchain-event-listener-service.js

```diff
@@ -140,6 +140,10 @@ class BlockchainEventListenerService {
         );
     }
 
+    fetchTokenIdsForParanet(paranetId) {
```

Comment:
@aleksamagicka Can you try to implement this function based on `getKnowledgeAssetsWithPagination` function in smart contracts which is available here: https://github.com/OriginTrail/dkg-evm-module/blob/6ede3739c0dd400aa6405e440ec01353f0febfb9/contracts/v1/storage/paranets/ParanetsRegistry.sol#L233